Portal/vip-portal/src/main/webapp/js/ami-viewer.js

```diff
@@ -2,23 +2,28 @@
 // https://codesandbox.io/s/github/FNNDSC/ami/tree/master/lessons/03
 
 function amiViewer(url, rawExtension, divId) {
+  window.setNoticeMessage(
+    'Loading and parsing image ' +
+      url.substring(url.lastIndexOf('/') + 1) +
+      '.  Please wait …')
```

Comment:
If I'm not mistaken, this is what takes most of the time. We should probably inform the user about it with a message like "Please wait, this may take a while..."

Reply:
It depends on the format of the image.  mhd/raw images are parsed faster than nifti images, for example.  For mhd/raw files, the most time is spent fetching the files from the SE.

But I don't mind changing the message.

Reply:
Ok, then let's use "Please wait, this may take a while..." for both messages (download and parsing)